.github/workflows/check.yaml

```diff
@@ -15,22 +15,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - uses: ./.github/actions/gradle-setup
 
       - name: Check Terraform files are properly formatted (run "terraform fmt -recursive" to fix)
         run: |
           terraform fmt -recursive
           git diff --exit-code
 
-      # run Checkstyle explicitly (as opposed to within gradle) due to better reporting capabilities
       - name: Run Checkstyle
```

Comment:
nit: I would suggest running this before `terraform fmt`, since by experience it fails much more often

Reply:
Applied. Thanks